tools/populate_dandiset_asset_yamls.py

```diff
@@ -68,14 +58,16 @@ def main():
                 }
             else:
                 metadata["path"] = str(relpath)
-                try:
-                    BareAssetMeta(**metadata)
-                except ValidationError as e:
-                    print(f"VALIDATION ERROR: {e}")
-                    with validation_errs.open("a") as fp:
-                        print(relpath, file=fp)
-                        traceback.print_exc(file=fp)
-                        print(file=fp)
+            errors = validate_file(f, schema_version=get_schema_version())
+            if errors:
+                print("VALIDATION ERRORS:")
```

Comment:
Yes, NWB errors and validation errors would end up in the same file.

Reply:
There's not much point in calling `validate_dandi_nwb()` from this script.  It would just call `nwb2asset()` again (the main portion of which, `get_metadata()`, is cached) and then capture any errors thrown by `BareAssetMeta(**asset.dict())`, with the end result being that conversion errors are mixed in with validation errors.  I think it would be better for the script to not call `validate_dandi_nwb()` and instead just do `BareAssetMeta(**asset.dict())` itself, like it did before.  The net effect would be no changes to `populate_dandiset_asset_yamls.py` and [some extra caching in dandi-cli](https://github.com/dandi/dandi-cli/pull/446).

Reply:
> There's not much point in calling validate_dandi_nwb() from this script.

I never suggested that ;-)  I am all for extra caching, merged that PR.  So we can drop this PR, right ? (I will also interrupt the run I initiated a few hours back)

Reply:
Yes, PR closed.